### PR TITLE
Updated chat design

### PR DESCRIPTION
**Pull request by markmiw**

### CODE REVIEW

File: client/dist/leaderboard.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.leaderboard-container {
+  overflow: ;
```
Comment (corneliusrenken) [inline client/dist/leaderboard.css:2]: never seen a container insecure about how it should overflow 🫣

**Review by corneliusrenken — APPROVED** [review]
bussin'

**Review by Jesse132 — APPROVED** [review]
Looks good, no foreseeable merge conflicts.

**Review by rixi-coding — APPROVED** [review]
Probably should have added DS.store in to gitignore XD

File: client/dist/leaderboard.css (hunk 1 above)
Comment (markmiw) [reply client/dist/leaderboard.css:2]: ill fix it later :)

**Comment by markmiw** [comment]
weird thing i have DS.store in gitignore :/

**Pull request merged**